### ISSUE TITLE
SQLiteStudio 3.4.6 will not launch in Linux (Fedora 41)

### ISSUE THREAD

**Issue by rebrinson** [pawelsalawa/sqlitestudio#5122]
### Details
3.4.4 launched fine and worked perfectly. I tried both installs (single user only and as admin) and both failed to launch. In the Linux App Menu, the Program reads:  /home/randy/SQLiteStudio/sqlitestudio  Any ideas?      ~lifecoachrandy@gmail.com

# Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
I'm not sure I understood you correctly. Are you saying, that entry in the App Menu points to the `/home/randy/SQLiteStudio/sqlitestudio`? Is this correct path f your installed SQLiteStudio? Have you verified it exists? If yes, then are you able to start it from shell (`./sqlitestudio` while being in `/home/randy/SQLiteStudio/`)?

**Comment by despairblue** [comment]
I think I have the same issue. I'm trying to start it from the CLI and it never opens a window. It also does not output debug logs:
![image](https://github.com/user-attachments/assets/b255b27d-234e-44c8-92aa-eade8530ec6f)

It is running though:
![image](https://github.com/user-attachments/assets/50672fef-c25c-4adf-a33e-328b20b308a5)

This is on sway/wayland, maybe related to that. I haven't tried on X.Org.

**Comment by pawelsalawa** [comment]
Please try with:
```
sqlitestudio -d --debug-stdout
```

By default the `-d` option keeps debug messages in the dedicated UI debug console. Since you have no access to UI of the application, we can redirect messages to stdout by adding `--debug-stdout`.

**Comment by despairblue** [comment]
Oh, I assumed `--debug-stdout` redirects from stderr to stdout 🤦🏾 

<details>
<summary>Logs</summary>

```
[15:31:00.914] DEBUG:    Using configuration directory: /home/despairblue/.config/SalSoft/SQLiteStudio
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/compress.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/csv.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/decimal.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/eval.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/icu.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/ieee754.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/percentile.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/rot13.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/series.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/sqlar.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/uint.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/uuid.so"
[15:31:00.915] DEBUG:    SQLite extension: "/usr/lib/sqlitestudio/extensions/zorder.so"
[15:31:00.948] DEBUG:    Following plugins found: ("PdfExport", "SqlEnterpriseFormatter", "XmlExport", "PythonSyntaxHighlighterPlugin", "MultiEditorImagePlugin", "DbAndroid", "CsvImport", "SqlFormatterSimplePlugin", "HtmlExport", "DbSqliteWx", "ScriptingPython", "DbSqliteCipher", "JsonExport", "SqlExport", "RegExpImport", "ConfigMigration", "Printing", "CsvExport", "ScriptingTcl")
[15:31:00.949] DEBUG:    PdfExport loaded: /usr/lib/sqlitestudio/plugins/libPdfExport.so
[15:31:00.951] DEBUG:    SqlEnterpriseFormatter loaded: /usr/lib/sqlitestudio/plugins/libSqlEnterpriseFormatter.so
[15:31:00.951] DEBUG:    XmlExport loaded: /usr/lib/sqlitestudio/plugins/libXmlExport.so
[15:31:00.951] DEBUG:    PythonSyntaxHighlighterPlugin loaded: /usr/lib/sqlitestudio/plugins/libPythonSyntaxHighlighter.so
[15:31:00.951] DEBUG:    MultiEditorImagePlugin loaded: /usr/lib/sqlitestudio/plugins/libMultiEditorImage.so
[15:31:00.952] DEBUG:    CsvImport loaded: /usr/lib/sqlitestudio/plugins/libCsvImport.so
[15:31:00.952] DEBUG:    HtmlExport loaded: /usr/lib/sqlitestudio/plugins/libHtmlExport.so
[15:31:00.953] DEBUG:    DbSqliteWx loaded: /usr/lib/sqlitestudio/plugins/libDbSqliteWx.so
[15:31:00.953] DEBUG:    DbSqliteCipher loaded: /usr/lib/sqlitestudio/plugins/libDbSqliteCipher.so
[15:31:00.954] DEBUG:    JsonExport loaded: /usr/lib/sqlitestudio/plugins/libJsonExport.so
[15:31:00.954] DEBUG:    SqlExport loaded: /usr/lib/sqlitestudio/plugins/libSqlExport.so
[15:31:00.954] DEBUG:    RegExpImport loaded: /usr/lib/sqlitestudio/plugins/libRegExpImport.so
[15:31:00.955] DEBUG:    ConfigMigration loaded: /usr/lib/sqlitestudio/plugins/libConfigMigration.so
[15:31:00.956] DEBUG:    Printing loaded: /usr/lib/sqlitestudio/plugins/libPrinting.so
[15:31:00.957] DEBUG:    CsvExport loaded: /usr/lib/sqlitestudio/plugins/libCsvExport.so
[15:31:00.958] DEBUG:    ScriptingTcl loaded: /usr/lib/sqlitestudio/plugins/libScriptingTcl.so
[15:31:00.959] DEBUG:    Loading form file: :/forms/CsvExport.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/CsvImportOptions.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/htmlexport.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/jsonexport.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/pdfexport.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populateconstant.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populatedictionary.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populaterandom.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populaterandomtext.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populatescript.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/plugins/populatesequence.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/regexpimport.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/sqlenterpriseformatter.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/SqlExportCommon.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/SqlExportQuery.ui
[15:31:00.959] DEBUG:    Loading form file: :/forms/XmlExport.ui
[15:31:00.984] WARNING:  Failed to load client buffer integration: "wayland-egl"
[15:31:00.984] WARNING:  Available client buffer integrations: ()
[15:31:00.985] WARNING:  No shell integration named "xdg-shell" found
[15:31:00.985] WARNING:  No shell integration named "xdg-shell-v6" found
[15:31:00.985] WARNING:  No shell integration named "wl-shell" found
[15:31:00.985] WARNING:  No shell integration named "ivi-shell" found
[15:31:00.985] WARNING:  Loading shell integration failed.
[15:31:00.985] WARNING:  Attempted to load the following shells ("xdg-shell", "xdg-shell-v6", "wl-shell", "ivi-shell")
[15:31:00.986] DEBUG:    Error from notify manager: "Could not add database .n8n/database.sqlite: Could not open database: unable to open database file; Could not open database: unable to open database file; Could not open database: unable to open database file"
[15:31:00.986] DEBUG:    Error from notify manager: "Could not add database .n8n/database.sqlite to list."
```

</details>

**Comment by pawelsalawa** [comment]
Okay, that helps. I will try to fix it, although it doesn't look easy to me.

**Comment by pawelsalawa** [comment]
A quick walkaround for now may be to use:
```
export QT_QPA_PLATFORM=xcb
sqlitestudio
```
On my Fedora 41 this works. Still I will try to solve the wayland issue.

**Comment by pawelsalawa** [comment]
I managed to fix it for upcoming 3.4.7.

**Comment by despairblue** [comment]
That makes sense, I had the same issue with avidemux and had to change the platform there too.

Thanks for fixing it ❤️
